saleor/graphql/product/tests/test_variant.py

```diff
@@ -6083,3 +6083,77 @@ def test_product_variant_deactivate_preorder_as_app(
     )
 
     assert_no_permission(response)
+
+
+VARIANT_CREATE_MUTATION = """
+    mutation variantCreate($productId: ID!, $attrId: ID!) {
+        productVariantCreate (
+            input: {
+                sku: "my-sku",
+                product: $productId,
+                attributes: [{id: $attrId, values:["1"]}]
```

Comment:
IMO we should pass here variables like `attributes: $variantAttrs`, to be able to reuse it in all tests.